01week/resume/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Jason Garcia - Resume</title>
+ <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <header>
+    <h1><center>Jason Garcia</h1></center>
+      <p><center><strong>E-Mail:</strong><a href="mailto:jasontx@gmail.com">jasontx@gmail.com</a>
+      <strong>Phone:</strong>210-417-6654
+      <strong>Location:</strong> Austin, TX
+      </p></center>
+  </header>
+
+  <nav><center>
```

Comment:
the <center> tag is an old HTML 4 tag is no longer used in HTML 5. Use CSS to manipulate text alignment.